Language/Haskell/GhcMod/ImportedFrom.hs

```diff
@@ -282,7 +282,7 @@ ghcPkgFindModule mod = do
 
     shortcut [ stackGhcPkgFindModule rp mod
              , hcPkgFindModule       rp mod
-             , _ghcPkgFindModule     rp mod
```

Comment:
FYI in GHC a leading underscore denotes a typed hole but you can also use it to suppress the "unused symbol" warning when you then go on to define that symbol. I'm not aware of what "python style" you refer to here.